ribs/emitters/rankers/_two_stage_random_direction_ranker.py

```diff
@@ -0,0 +1,73 @@
+"""
+Provides the TwoStageRandomDirectionRanker
+"""
+import numpy as np
+
+from ribs.emitters.rankers._ranker_base import RankerBase
+
+
+class TwoStageRandomDirectionRanker(RankerBase):
```

Comment:
Should have an init function here which creates the `self._target_behavior_dir` property.

Reply:
And sets it to None